src/game_api/level_api.hpp

```diff
@@ -462,3 +466,280 @@ void add_next_levels(std::vector<std::string> next_levels);
 
 int8_t get_co_subtheme();
 void force_co_subtheme(int8_t subtheme);
+
+class CustomTheme : public ThemeInfo
+{
+  public:
+    uint8_t theme = 100;
```

Comment:
This list is a clusterfuck, but the idea is you can disable individual aspects of the level gen or get them from a different base theme